xonsh/commands_cache.py

```diff
@@ -99,11 +99,11 @@ def all_commands(self):
             # entries at the back.
             for cmd in executables_in(path):
                 key = cmd.upper() if ON_WINDOWS else cmd
-                allcmds[key] = (os.path.join(path, cmd), cmd in alss)
+                allcmds[key] = (os.path.join(path, cmd), alss.get(key, None))
         for cmd in alss:
             if cmd not in allcmds:
                 key = cmd.upper() if ON_WINDOWS else cmd
-                allcmds[key] = (cmd, True)
+                allcmds[key] = (cmd, None)
```

Comment:
Ahh, that's an interesting point. In this case, there's an alias that doesn't exist in `allcmds` for some reason -- the second value of the tuple is (now) the name of the alias if it exists and the first value is the full string path of the command, so in this case, the "alias" value is the same as the cmd value. I just set it to None, but it could be `(cmd, cmd)`